src/host/ut_host/AliasTests.cpp

```diff
@@ -181,258 +180,4 @@ class AliasTests
         VERIFY_IS_TRUE(buffer.empty());
         VERIFY_ARE_EQUAL(1u, dwLines);
     }
-
-    TEST_METHOD(Tokenize)
-    {
-        std::wstring tokenStr(L"one two three");
-        std::deque<std::wstring> tokensExpected;
-        tokensExpected.emplace_back(L"one");
-        tokensExpected.emplace_back(L"two");
-        tokensExpected.emplace_back(L"three");
-
-        auto tokensActual = Alias::s_Tokenize(tokenStr);
-
-        VERIFY_ARE_EQUAL(tokensExpected.size(), tokensActual.size());
-
-        for (size_t i = 0; i < tokensExpected.size(); i++)
-        {
-            VERIFY_ARE_EQUAL(String(tokensExpected[i].data()), String(tokensActual[i].data()));
-        }
-    }
-
-    TEST_METHOD(TokenizeNothing)
-    {
-        std::wstring tokenStr(L"alias");
-        std::deque<std::wstring> tokensExpected;
-        tokensExpected.emplace_back(tokenStr);
-
-        auto tokensActual = Alias::s_Tokenize(tokenStr);
-
-        VERIFY_ARE_EQUAL(tokensExpected.size(), tokensActual.size());
-
-        for (size_t i = 0; i < tokensExpected.size(); i++)
-        {
-            VERIFY_ARE_EQUAL(String(tokensExpected[i].data()), String(tokensActual[i].data()));
-        }
-    }

```

Comment:
Tokenization doesn't exist anymore as a separate function. The `megamix` test above is probably sufficient to cover tokenization however.